src/AppInstallerRepositoryCore/Rest/Schema/1_0/Interface.cpp

```diff
@@ -129,13 +130,14 @@ namespace AppInstaller::Repository::Rest::Schema::V1_0
 
     IRestClient::SearchResult Interface::OptimizedSearch(const SearchRequest& request) const
     {
-        // TODO: Send in VersionLatest = true query param.
         SearchResult searchResult;
-        std::vector<Manifest::Manifest> manifests = GetManifests(request.Filters[0].Value, {}, {});
+        std::map<std::string_view, std::string> queryParams;
+        queryParams.emplace(LatestVersionQueryParam, "true");

```

Comment:
>queryParams.emplace(LatestVersionQueryParam, "true"); [](start = 8, length = 53)

Per-offline discussions, we want to remove this.  And I don't think it is actually appropriate here anyway, as we don't know the caller's intention RE: versions at this point.  I know I was the one who said to add it, but I can be wrong.